docker-compose/local/django/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-alpine
+FROM python:3.8-alpine
```

Comment:
I'd love to move this to Ubuntu to match production at some point, if it isn't hard to do.

Reply:
It shouldn't be that hard.